mindsdb/integrations/utilities/rag/pipelines/rag.py

```diff
@@ -9,8 +10,6 @@
 from mindsdb.integrations.utilities.rag.settings import RAGPipelineModel, DEFAULT_AUTO_META_PROMPT_TEMPLATE
 from mindsdb.integrations.utilities.rag.vector_store import VectorStoreOperator
 
-from mindsdb.interfaces.agents.safe_output_parser import SafeOutputParser
```

Comment:
This was the main issue behind the problem with streaming, streaming agent executor in langchain expects message to be a string not a dictionary.